package-scripts.js

```diff
@@ -95,8 +101,30 @@ module.exports = {
       default: npsUtils.series.nps("lint", "test")
     },
     watch: {
-      es: "lerna exec --parallel --ignore victory-native --ignore victory-vendor -- cross-env BABEL_ENV=es babel src --out-dir es --config-file ../../.babelrc.build.js --copy-files --extensions .tsx,.ts,.jsx,.js --watch",
-      lib: "lerna exec --parallel --ignore victory-native --ignore victory-vendor -- cross-env BABEL_ENV=commonjs babel src --out-dir lib --config-file ../../.babelrc.build.js --copy-files --extensions .tsx,.ts,.jsx,.js --watch",
+      debug:
+        'lerna exec --scope victory-area -- nps \\"types.copy es --watch\\"',
+      "types-create-lib":
+        'lerna exec --parallel --ignore victory-native --ignore victory-vendor -- nps \\"types.create --outDir lib --watch\\"',
+      "types-create-es":
+        'lerna exec --parallel --ignore victory-native --ignore victory-vendor -- nps \\"types.create --outDir es --watch\\"',
+      "types-copy-lib":
+        'lerna exec --parallel --ignore victory-native --ignore victory-vendor -- nps \\"types.copy lib --watch\\"',
+      "types-copy-es":
+        'lerna exec --parallel --ignore victory-native --ignore victory-vendor -- nps \\"types.copy es --watch\\"',
+      "babel-es":
+        "lerna exec --parallel --ignore victory-native --ignore victory-vendor -- cross-env BABEL_ENV=es babel src --out-dir es --config-file ../../.babelrc.build.js --extensions .tsx,.ts,.jsx,.js --watch",
+      "babel-lib":
+        "lerna exec --parallel --ignore victory-native --ignore victory-vendor -- cross-env BABEL_ENV=commonjs babel src --out-dir lib --config-file ../../.babelrc.build.js --extensions .tsx,.ts,.jsx,.js --watch",
+      lib: npsUtils.concurrent.nps(
+        "watch.babel-lib",
+        "watch.types-create-lib",
+        "watch.types-copy-lib"
+      ),
```

Comment:
This seems to be working fine on my machine.  It takes a while to get started, but after a minute the CPU goes back to normal while waiting for changes.

However, I was going to make a suggestion.  It might be nice to reorganize these scripts a little bit, so that I can run "watch" for just 1 specific patckage (eg. `victory-core`)
For example, I would like to run:
```
lerna exec --scope victory-core "nps watch.lib"
```
and then maybe we could have a single `watch.default` which runs `lerna exec --parallel ...`

What do you think of this approach?

Reply:
Done :) It's very easy now to run for specific packages!